src/main/java/jnr/posix/BaseNativePOSIX.java

```diff
@@ -809,4 +809,10 @@ public int getgroups(int size, long[] groups) {
     public String nl_langinfo(int item) {
         return libc().nl_langinfo(item);
     }
+
+    @Override
+    public String strerror(int code)
+    {
```

Comment:
If not too much of a hassle, please use the brace style the rest of the code base uses. Unfortunately, we don't have anything to reformat the code on merge.